docs/manual.rst

```diff
@@ -1097,9 +1097,35 @@ differently.
   >>> b == c
   False
 
-.. method:: object.equals_exact(other, tolerance)
-
-  Returns ``True`` if the object is within a specified `tolerance`.
+.. method:: object.equals_exact(other, tolerance=0.0, normalize=False)
```

Comment:
I think that no additions to `manual.rst` are required since the docs will be generated from the docstrings of `shapely/predicates.py`